keps/NNNN-kep-template/README.md

```diff
@@ -270,6 +266,55 @@ when drafting this test plan.
 [testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
 
+[ ] I/we understand the owners of the involved components may require updating
+existing tests to make this code solid enough prior committing changes necessary
+to implement this enhancement.
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+<!--
+In principle every added code should be unit tested. However, the exact tests are hard
+to answer ahead of implementation. As a result, this section should only be filled when
+targeting Beta to ensure that something wasn't missed during Alpha implementation.
+
+Talking about individual tests would be an overkill, so we just require to list the
+packages that were touched during the implementation together with their current test
+coverage in the form:
+- <package>: <current test coverage>
+
+The data can be easily read from:
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+--->
+
+- <package>: <test coverage>
+
+##### Integration tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
```

Comment:
same form as above:
- test: link to test coverage

Reply:
done